README.md

```diff
@@ -95,16 +94,10 @@ Hermann is the kafka client library that you will need to explicitly install if
 Caveat: Hermann is only usable from within Jruby, due to its implementation of zookeeper based broker discovery being JVM based.
 
 ```ruby
-# zipkin-kafka-tracer requires Hermann 0.25.0 or later
-gem 'hermann', '~> 0.25'
+# zipkin-kafka-tracer requires Hermann > 0.26.1.0
+gem 'hermann', '> 0.26.1.0'
```

Comment:
such a gem doesn't exist yet, waiting for the hermann people to cut a new version...